Koronavilkku/Koronavilkku/ViewController/NotificationListViewController.swift

```diff
@@ -0,0 +1,197 @@
+                var bottom: ConstraintItem? = nil
+                
+                for notification in notifications.sorted(by: { $0.detectedOn > $1.detectedOn }) {
+                    let item = self.createNotificationItem(notification: notification)
+                    top = self.notificationListWrapper.appendView(item, spacing: 20, top: top)
+                    bottom = item.snp.bottom
+                }
+                
+                if let bottom = bottom {
+                    self.notificationListWrapper.snp.makeConstraints { make in
+                        make.bottom.equalTo(bottom)
+                    }
+                }
+            }
+            .store(in: &updateTasks)
+        
+        exposureRepository.timeFromLastCheck()
+            .sink { [weak self] time in
+                self?.lastCheckedView.timeFromLastCheck = time
+            }
+            .store(in: &updateTasks)
+    }
+    
+    func createNotificationItem(notification: ExposureNotification) -> CardElement {
+        CardElement().layout { append in
+            let dateFormatter = DateFormatter()
+            dateFormatter.setLocalizedDateFormatFromTemplate("dMyyyy")
+
+            let detectionDate = dateFormatter.string(from: notification.detectedOn)
+            let titleView = UILabel(label: Text.ItemTitle.localized(with: detectionDate),
+                                    font: .heading4,
+                                    color: UIColor.Greyscale.black)
+            titleView.numberOfLines = 0
+            
+            append(titleView,
+                   .init(top: 20, left: 20, right: 20))
+            
+            append(.createDivider(height: 1),
+                   .init(top: 10, left: 20, right: 20))
+            
+            append(createLineItem(label: .ItemCountLabel,
+                                  valueText: .ItemCountValue,
+                                  value: String(notification.exposureCount)),
+                   .init(top: 10, left: 20, right: 20))
+            
+            let intervalFormatter = DateIntervalFormatter()
+            intervalFormatter.timeStyle = .none
+            let interval = intervalFormatter.string(from: notification.detectionInterval.start,
+                                                    to: notification.detectionInterval.end)
+            append(createLineItem(label: .ItemIntervalLabel,
+                                  valueText: .ItemIntervalValue,
+                                  value: interval),
+                   .init(top: 10, left: 20, bottom: 20, right: 20))
+        }
+    }
+    
+    private func createLineItem(label: Text, valueText: Text, value: CVarArg) -> UIView {
+        let line = UIView()
+        
+        let label = UILabel(label: label.localized,
+                            font: .bodySmall,
+                            color: UIColor.Greyscale.black)
+        
+        label.numberOfLines = 0
```

Comment:
Noticed a strange issue with the layout: on iPhone 11 (13.6) if I enable large fonts from accessibility settings and select the 3rd largest font, then the value text isn't fully visible (see the first screenshot), but with the 2nd both label and value are nicely visible. 

![strange](https://user-images.githubusercontent.com/66993353/99994777-22760f80-2dc2-11eb-8c7f-2d92196e3610.png)
![ok](https://user-images.githubusercontent.com/66993353/99994781-243fd300-2dc2-11eb-9dea-05426457d9ea.png)

Reply:
I'll try to adjust the parameters a bit…